packages/trader/src/AppV2/Containers/ContractDetails/contract-details.scss

```diff
@@ -11,7 +11,7 @@ $TWO_BUTTONS_FOOTER_HEIGHT: 120px;
     overflow-y: auto;
     overflow-x: hidden;
     gap: var(--semantic-spacing-general-sm);
-    height: 100%;
+    height: calc(100vh - 40px);
```

Comment:
are we trying to subtract footer height ?

Reply:
Yes.